.github/workflows/release.yaml

```diff
@@ -43,7 +44,7 @@ jobs:
 
     - name: Output branch name
       id: branch
-      run: echo "::set-output name=branch_name::$BRANCH_NAME"
+      run: echo "{branch_name}={$BRANCH_NAME}" >> $GITHUB_OUTPUT
```

Comment:
Haas this been tested on a fork or another workflow?

Reply:
No, but it is taken directly [from the GitHub docs for migrating](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) away from `::set-output` which no longer works.